u17_ithaca.xml

```diff
@@ -831,7 +830,7 @@ in <placeName type="street"><place>O'Connell street lower</place></placeName>,
 <lb n="170826"/><l>And cut off his little head.</l>
 <lb n="170827"/><l>And now he'll play his ball no more</l>
 <lb n="170828"/><l>For he lies among the dead.</l></lg></quote>
-    [MUSIC]
+<p rend="non-indent"><lb n="170828b"/>[MUSIC]</p>
```

Comment:
This was the little bugger breaking Travis originally. 

(Then my `<place>` tags broke it times a hundred...)